package.json

```diff
@@ -52,8 +52,8 @@
     }
   ],
   "dependencies": {
-    "@particle/device-constants": "^3.5.0",
-    "binary-version-reader": "^2.5.1",
+    "@particle/device-constants": "^3.6.0",
+    "binary-version-reader": "^2.5.2",
```

Comment:
We shouldn't bump binary-version-reader when updating device-constants. The point of having device-constants as a peer dependency in binary-version-reader is to avoid updating dependencies transiently. Only the top-level application has to update device-constants.

No need to undo it here, but no need to update in other applications.